config/cesm/machines/config_machines.xml

```diff
@@ -428,42 +428,29 @@ This allows using a different mpirun command to launch unit tests
         <command name="load">gnu/9.1.0</command>
         <command name="load">openblas/0.3.6</command>
       </modules>
-      <modules compiler="intel" mpilib="!mpi-serial" DEBUG="TRUE" comp_interface="mct">
-        <command name="load">esmf-8.0.0-defio-mpi-g</command>
-      </modules>
-      <modules compiler="intel" mpilib="!mpi-serial" DEBUG="FALSE" comp_interface="mct">
-        <command name="load">esmf-8.0.0-defio-mpi-O</command>
-      </modules>
-      <modules compiler="intel" mpilib="mpi-serial" DEBUG="TRUE" comp_interface="mct">
-        <command name="load">esmf-8.0.0-ncdfio-uni-g</command>
-      </modules>
-      <modules compiler="intel" mpilib="mpi-serial" DEBUG="FALSE" comp_interface="mct">
-        <command name="load">esmf-8.0.0-ncdfio-uni-O</command>
+      <modules compiler="pgi">
+	<command name="load">pgi/19.3</command>
       </modules>
 
-      <modules compiler="intel" mpilib="!mpi-serial" DEBUG="TRUE" comp_interface="nuopc">
+      <modules compiler="intel" mpilib="!mpi-serial" DEBUG="TRUE">
         <command name="use">/glade/work/turuncu/PROGS/modulefiles/esmfpkgs/intel/19.0.5</command>
         <command name="load">esmf-8.1.0b14-ncdfio-mpt-g</command>
       </modules>
-
-      <modules compiler="gnu" mpilib="!mpi-serial" DEBUG="TRUE" comp_interface="nuopc">
-        <command name="use">/glade/work/turuncu/PROGS/modulefiles/esmfpkgs/gnu/9.1.0</command>
-        <command name="load">esmf-8.1.0b14-ncdfio-mpt-g</command>
-      </modules>
-      <modules compiler="intel" mpilib="!mpi-serial" DEBUG="FALSE" comp_interface="nuopc">
+      <modules compiler="intel" mpilib="!mpi-serial" DEBUG="FALSE">
         <command name="use">/glade/work/turuncu/PROGS/modulefiles/esmfpkgs/intel/19.0.5</command>
         <command name="load">esmf-8.1.0b14-ncdfio-mpt-O</command>
       </modules>
-      <modules compiler="intel" mpilib="mpi-serial" DEBUG="TRUE" comp_interface="nuopc">
+      <modules compiler="intel" mpilib="mpi-serial" DEBUG="TRUE">
         <command name="use">/glade/work/turuncu/PROGS/modulefiles/esmfpkgs/intel/19.0.5</command>
         <command name="load">esmf-8.1.0b14-ncdfio-mpiuni-g</command>
       </modules>
-      <modules compiler="intel" mpilib="mpi-serial" DEBUG="FALSE" comp_interface="nuopc">
+      <modules compiler="intel" mpilib="mpi-serial" DEBUG="FALSE">
         <command name="use">/glade/work/turuncu/PROGS/modulefiles/esmfpkgs/intel/19.0.5</command>
         <command name="load">esmf-8.1.0b14-ncdfio-mpiuni-O</command>
       </modules>
-      <modules compiler="pgi">
-	<command name="load">pgi/19.3</command>
+      <modules compiler="gnu" mpilib="!mpi-serial" DEBUG="TRUE">
+        <command name="use">/glade/work/turuncu/PROGS/modulefiles/esmfpkgs/gnu/9.1.0</command>
+        <command name="load">esmf-8.1.0b14-ncdfio-mpt-g</command>
       </modules>
```

Comment:
The latest gnu version of esmf I could find was esmf-8.1.0b11-ncdfio-mpt-g.  Could you also add DEBUG="FALSE" for gnu esmf-8.1.0b11-ncdfio-mpt-O, just in case it's ever needed.

Reply:
Will this do for now?  Please open an issue asking for esmf builds for gnu and pgi compilers and assign to @uturuncoglu

Reply:
ESMF builds on Cheyenne should no longer be handled by @uturuncoglu -- this is now responsibility of @him-28.

Reply:
Has @him-28 been added as a contributor to cime, I can't assign issue #3496 to him.